pints/_log_priors.py

```diff
@@ -95,6 +97,93 @@ def sample(self, n=1):
         return np.random.beta(self._a, self._b, size=(n, 1))
 
 
+class BinomialLogPrior(pints.LogPrior):
+    r"""
+    Defines a binomial (log) prior with given number of trials parameter
+    ``trials`` and trial success probability parameter ``prob`` with
+    pdf
+
+    .. math::
+        f(x|\text{trials}, \text{prob}) = {\text{trials} \choose x}\;
+        \text{prob}^{x}\;(1-\text{prob})^(\text{trials}-x).
+
+    A random variable :math:`X` distributed according to this pdf has
+    expectation
+
+    .. math::
+        \mathrm{E}(X)=\text{trials} \; \text{prob}.
+
+    For example, to create a prior with ``trials=10`` and ``prob=0.5``
+    use::
+
+        p = pints.BinomialLogPrior(10, 0.5)
+
+    Extends :class:`LogPrior`.
+    """
+
+    def __init__(self, trials, prob):
+        # Parse input arguments
+        self._trials = float(trials)
+        self._prob = float(prob)
+
+        # Validate inputs
+        if not self._trials.is_integer():
+            raise TypeError('Count parameter "trials" must be \
+                integer.')
```

Comment:
```suggestion
            raise TypeError('Count parameter "trials" must be'
                            ' integer.')
```

Reply:
And same for the others below (otherwise you add lots of whitespace in the middle of the string)

There's a special rule for strings that says you can write

```
print('hello'
      'there')
```
instead of
```
print('hello'
      + 'there')
```

though both are valid